site/docs/tasks/using-opa.md

```diff
@@ -47,3 +69,19 @@ kubectl label configmap <policy-name> openpolicyagent.org/policy=rego -n fybrik-
 ```
 
 Delete the policy with `kubectl delete configmap <policy-name> -n fybrik-system`.
+
+
+### Using `opaServer.bootstrapPolicies` field
+
+Another method to upload policies to OPA is to write them in `opaServer.bootstrapPolicies` field in [values.yaml](https://raw.githubusercontent.com/fybrik/charts/master/charts/fybrik/values.yaml) file used for the Fybrik helm chart deployment.
```

Comment:
in field -> as field

Reply:
Fybrik helm chart deployment -> Fybrik deployment

Reply:
Fix. Thanks